sample/.azureFunctions/swagger/swagger.json

```diff
@@ -0,0 +1,1046 @@
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "id",
+                        "in": "path",
+                        "required": true,
+                        "type": "string"
+                    }
+                ],
+                "description": "Replace with Operation Object #http://swagger.io/specification/#operationObject",
+                "responses": {
+                    "200": {
+                        "description": "Success operation"
+                    }
+                },
+                "security": [
+                    {
+                        "apikeyQuery": [ ]
+                    }
+                ]
+            },
+            "post": {
+                "operationId": "/api/node/products/{category}/{id}/post",
+                "produces": [ ],
+                "consumes": [ ],
+                "parameters": [
+                    {
+                        "name": "category",
+                        "in": "path",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "id",
+                        "in": "path",
+                        "required": true,
+                        "type": "string"
+                    }
+                ],
+                "description": "Replace with Operation Object #http://swagger.io/specification/#operationObject",
+                "responses": {
+                    "200": {
+                        "description": "Success operation"
+                    }
+                },
+                "security": [
+                    {
+                        "apikeyQuery": [ ]
+                    }
+                ]
+            },
+            "put": {
+                "operationId": "/api/node/products/{category}/{id}/put",
+                "produces": [ ],
+                "consumes": [ ],
+                "parameters": [
+                    {
+                        "name": "category",
+                        "in": "path",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "id",
+                        "in": "path",
+                        "required": true,
+                        "type": "string"
+                    }
+                ],
+                "description": "Replace with Operation Object #http://swagger.io/specification/#operationObject",
+                "responses": {
+                    "200": {
+                        "description": "Success operation"
+                    }
+                },
+                "security": [
+                    {
+                        "apikeyQuery": [ ]
+                    }
+                ]
+            }
+        },
+        "/api/HttpTrigger-Disabled": {
```

Comment:
this was generated from HttpRouteCollection. Will have to add a mechanism to exclude the disabled functions.